docs/howto/create-a-workspace.md

```diff
@@ -0,0 +1,28 @@
+# Creating a Workspace
+A Workspace is a directory containing information about one or more SDK produced plugins to be packaged into a single, deployable `.yxi` archive. A given Workspace contains a `ayx_workspace.json`, and all other necessary components to build plugins.

```

Comment:
TW review:

A Workspace is a directory that contains information about one or more SDK-produced plugins to be packaged into a single, deployable `.yxi` archive. A given Workspace contains an `ayx_workspace.json` file and all other necessary components to build plugins.